packages/entity-list/src/main.js

```diff
@@ -105,12 +105,13 @@ class EntityListApp extends React.Component {
     this.app = initApp(props.id, props, events)
   }
 
-  componentWillReceiveProps = nextProps => {
-    const changedProps = _pickBy(nextProps, (value, key) => !_isEqual(value, this.props[key]))
-
-    getDispatchActions(changedProps).forEach(action => {
-      this.app.store.dispatch(action)
-    })
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (!_isEqual(prevProps, this.props)) {
```

Comment:
`  if (!_isEqual(prevProps, this.props)) `
Needed? On the next line you allready check for changes and please keep the object-dif helper method as shown to you in mind